### PR TITLE
de+en: Enhance current privacy revision

### PR DESCRIPTION
**Pull request by kulla**
This PR includes first enhancements made in https://docs.google.com/document/d/1FmHyYlRBKjIhnmYbdzdJFgQ-7BvHbeJOtcSqXMsv8Bc/edit?usp=sharing

### CODE REVIEW

File: de/privacy/current.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+Cookies, die zur Durchführung des elektronischen Kommunikationsvorgangs oder zur Bereitstellung bestimmter, von Dir erwünschter Funktionen erforderlich sind, werden auf Grundlage von Art. 6 Abs. 1 lit. f DSGVO gespeichert. Unser berechtigtes Interesse an der Speicherung von Cookies liegt in der Ermöglichung einer  technisch fehlerfreien und optimierten Bereitstellung unserer Dienste. Soweit andere Cookies (z.B. Cookies zur Analyse Deines Surfverhaltens) gespeichert werden, werden diese in dieser Datenschutzerklärung gesondert behandelt.
 
 #### Server-Log-Dateien
 
-Der Provider der Seiten erhebt und speichert automatisch Informationen in so genannten Server-Log-Dateien, die Dein Browser automatisch an uns übermittelt. Dies sind:
+Wir als Provider der Webseiten der Seiten erheben und speichern automatisch Informationen in so genannten Server-Log-Dateien, die Dein Browser automatisch an uns übermittelt. Dies sind:
```
Comment (inyono) [inline de/privacy/current.md:30]:
```suggestion
Wir als Provider der Webseiten erheben und speichern automatisch Informationen in so genannten Server-Log-Dateien, die Dein Browser automatisch an uns übermittelt. Dies sind:
```

File: de/privacy/current.md [hunk 2]
```diff
@@ -99,15 +99,17 @@ Die gesammelten Informationen umfassen:
...
 
-Unsere Website nutzt Pldas ugins der Website YouTube. Betreiber der Seiten ist die YouTube, LLC, 901 Cherry Ave., San Bruno, CA 94066, USA.
+Unsere Website nutzt Plugins der Website YouTube. Betreiber der Seiten ist die YouTube, LLC, 901 Cherry Ave., San Bruno, CA 94066, USA.
 
-Wir nutzen YouTube im erweiterten Datenschutzmodus. Dieser Modus bewirkt laut YouTube, dass YouTube keine Informationen über die BesucherInnen auf dieser Website speichert, bevor diese sich das Video ansehen. Die Weitergabe von Daten an YouTube-Partner wird durch den erweiterten Datenschutzmodus hingegen nicht zwingend ausgeschlossen. So stellt YouTube – unabhängig davon, ob Du dir ein Video ansiehst – eine Verbindung zum Google DoubleClick-Netzwerk her.
+Wir nutzen YouTube im erweiterten Datenschutzmodus. Dieser Modus bewirkt, dass YouTube keine Informationen über die Besucher*innen auf dieser Webseite speichert, bevor diese sich das Video ansehen. Die Weitergabe von Daten an YouTube-Partner wird durch den erweiterten Datenschutzmodus hingegen nicht zwingend ausgeschlossen. So stellt YouTube – unabhängig davon, ob Du dir ein Video ansiehst – eine Verbindung zum Google DoubleClick-Netzwerk her.
```
Comment (inyono) [inline de/privacy/current.md:112]:
```suggestion
Wir nutzen YouTube im erweiterten Datenschutzmodus. Dieser Modus bewirkt, dass YouTube keine Informationen über die Besucher\*innen auf dieser Webseite speichert, bevor diese sich das Video ansehen. Die Weitergabe von Daten an YouTube-Partner wird durch den erweiterten Datenschutzmodus hingegen nicht zwingend ausgeschlossen. So stellt YouTube – unabhängig davon, ob Du dir ein Video ansiehst – eine Verbindung zum Google DoubleClick-Netzwerk her.
```

**Comment by kulla** [comment]
> Noted some things. We still should create a new revision (that also includes the changes you are going to discuss with Miriam).

You mean I should continue my work with Miriam in this PR?

**Comment by inyono** [comment]
> You mean I should continue my work with Miriam in this PR?

Yes, so that we don't publish multiple privacy revisions in a short timeframe.

**Comment by kulla** [comment]
@inyono The necessary changes for the privacy revision in order to support the new frontend are added. This PR can be merged.

File: de/privacy/current.md [hunk 3]
```diff
@@ -69,25 +69,43 @@ Diese Website nutzt Funktionen des Webanalysedienstes Google Analytics. Anbieter
...
 
 Die Datenschutzrichtlinien von Google findest Du [hier](https://policies.google.com/?hl=de). Weitere Informationen dazu, wie Google Ihre Daten verarbeitet findest Du [hier](https://support.google.com/analytics/answer/6004245?hl=en&ref_topic=2919631).
 
+Die Benutzung erfolgt auf Grundlage von Art. 6 Abs. 1 lit. f. DSGVO. Unser berechtigtes Interesse an der Analyse des Nutzerverhaltens liegt in der Ermöglichung der Verbesserung unserer Websangebote.
+
+#### Matamo
```
Comment (inyono) [inline de/privacy/current.md:88]:
```suggestion
#### Matomo
```

File: de/privacy/current.md [hunk 4]
```diff
@@ -69,25 +69,43 @@ Diese Website nutzt Funktionen des Webanalysedienstes Google Analytics. Anbieter
...
+* das verwendete Betriebssystem,
+* die Referrer-URL (die zuvor besuchte Seite),
+* die IP-Adresse in anonymisierter Form
+* Sprach- und Ländereinstellungen
+* Anzahl und Verweildauer der Besuche
+* auf der Website betätigten Links
+* die Uhrzeit der Serveranfrage
+* Die von Matomo gespeicherten Daten werden nicht auf andere Server übertragen oder an Dritte weitergegeben.
+
+Matomo ist auf unserer Webseite so konfiguriert, dass eine im Browser vorgenommene »Do‑Not‑Track«-Einstellung beachtet wird. Wenn Du mit der Speicherung und Auswertung Deiner personenbezogenen Daten nicht einverstanden bist, aktiviere die »Do‑Not‑Track«-Einstellung Deines Browsers.
```
Comment (inyono) [inline de/privacy/current.md:102]: The used quotation marks should be consistent (see e.g. "demografische Merkmale")

File: de/privacy/current.md [hunk 5]
```diff
@@ -69,25 +69,43 @@ Diese Website nutzt Funktionen des Webanalysedienstes Google Analytics. Anbieter
...
+
```
Comment (inyono) [inline de/privacy/current.md:103]: Needs Matomo opt out iframe

File: de/privacy/current.md [hunk 6]
```diff
@@ -175,6 +210,42 @@ Diese Webseite nutzt die Google Cloud Plattform, ein Cloud Computing Service der
...
+* den Browser-Typ/-Version,
+* das verwendete Betriebssystem,
+* die Referrer-URL (die zuvor besuchte Seite),
+* die IP-Adresse,
+* die Uhrzeit der Serveranfrage
+* Cookies
+
+Diese Daten werden je nach Serverstandort auch in die USA übertragen. Wir haben mit Vercel einen Vertrag über die Auftragsverarbeitung abgeschlossen. Cloudflare ist unter dem [EU-US Datenschutzabkommen zertifiziert](https://www.privacyshield.gov/participant?id=a2zt0000000GnZKAA0&status=Active) und damit verpflichtet, den EU-Datenschutzvorgaben nachzukommen.
+
+Die Datenschutzerklärung von Vercel findest Du unter [https://www.cloudflare.com/security-policy](https://www.cloudflare.com/security-policy).
```
Comment (inyono) [inline de/privacy/current.md:228]:
```suggestion
Die Datenschutzerklärung von Cloudflare findest Du unter [https://www.cloudflare.com/security-policy](https://www.cloudflare.com/security-policy).
```

File: de/privacy/current.md [hunk 7]
```diff
@@ -175,6 +210,42 @@ Diese Webseite nutzt die Google Cloud Plattform, ein Cloud Computing Service der
...
+Diese Webseite nutzt den Dienst [Vercel](https://vercel.com/). Der Anbieter ist Vercel Inc., 340 S Lemon Ave #4133, Walnut, CA 91789, USA. Vercel ist eine Cloudplattform, über die wir unsere Webseite bereitstellen. Das bedeutet, dass der Besuch unser Webseite durch die Server von Vercel bearbeitet oder durchgeleitet wird. Die mit deinem Besuch unserer Webseite verbundenen Daten werden auch an Vercel übermittelt. Dies ist notwendig, damit Deine Browseranfrage erfolgreich bearbeitet werden kann. Diese Daten umfassen:
+
+* den Browser-Typ/-Version,
+* das verwendete Betriebssystem,
+* die Referrer-URL (die zuvor besuchte Seite),
+* die IP-Adresse,
+* die Uhrzeit der Serveranfrage
+* Cookies
+
+Diese Daten werden je nach Serverstandort auch in die USA übertragen. Wir haben mit Vercel einen Vertrag über die Auftragsverarbeitung abgeschlossen. Außerdem ist Vercel unter dem [EU-US Datenschutzabkommen zertifiziert](https://www.privacyshield.gov/participant?id=a2zt0000000TTIbAAO&status=Active) und damit verpflichtet, den EU-Datenschutzvorgaben nachzukommen.
```
Comment (inyono) [inline de/privacy/current.md:243]:
```suggestion
Diese Daten werden je nach Serverstandort auch in die USA übertragen. Außerdem ist Vercel unter dem [EU-US Datenschutzabkommen zertifiziert](https://www.privacyshield.gov/participant?id=a2zt0000000TTIbAAO&status=Active) und damit verpflichtet, den EU-Datenschutzvorgaben nachzukommen.
```
Vercel doesn't have such thing

**Comment by kulla** [comment]
@inyono This PR is ready to be merged. Matomo opt out iframe is added and I replaced the `_` with `-` in the special variables since it has a special meaning in markdown. However https://github.com/serlo/serlo.org-cloudflare-worker/pull/12 needs to be merged simultaneously.

**Pull request merged**